shared/constants/transaction.js

```diff
@@ -11,7 +11,8 @@ import { MESSAGE_TYPE } from './app';
  * @property {'approve'} TOKEN_METHOD_APPROVE - A token transaction requesting an
  *  allowance of the token to spend on behalf of the user
  * @property {'incoming'} INCOMING - An incoming (deposit) transaction
- * @property {'sentEther'} SENT_ETHER - A transaction sending ether to a recipient
+ * @property {'sentEther'} SENT_ETHER - A transaction sending ether to a recipient (LEGACY - replaced by sendingNativeAsset)
+ * @property {'sendingNativeAsset'} SENDING_NATIVE_ASSET - A transaction sending a network's native asset to a recipient
```

Comment:
I do not like conflating transaction types with translation strings. I would prefer us to use something like 'simpleSend', and then use the sendingNativeAsset key when we encounter that type of transaction. I know we already conflate the others, but this is an opportunity to move in the right direction.

Reply:
Ok yeah that makes sense. I'll make this change.